contrib/keyboard_teleop/teleop_mocap.py

```diff
@@ -0,0 +1,232 @@
+import mujoco
```

Comment:
Can you run `ruff format` on this directory btw? I think the formatting here is slightly off.